### ISSUE TITLE
Bug - Apps intended for the Tray are shown in den Panel

### ISSUE THREAD

**Issue by egrath** [linuxdeepin/developer-center#2515]
Since the upgrade to 20.4 i noticed that the Nextcloud Application Icon is persistently shown in the Panel and in the Tray, where it only should display it's icon in the tray:

![image](https://user-images.githubusercontent.com/1955895/150284788-73c39c3b-cd15-4f43-a846-57656baf31dc.png)

As soon as you open the main application window of Nextcloud using the tray icon and close it afterwards, the icon in the panel vanishes.

**Comment by tsic404** [comment]
![image](https://user-images.githubusercontent.com/48748112/162104857-7c0cef52-839d-433d-bbe0-d79da721a8ed.png)
![image](https://user-images.githubusercontent.com/48748112/162104899-f5f2ff24-ea30-47f8-b971-7bf9b443a7fb.png)
Similar to the picture above, do you mean after clicking the X, it still shown in task bar?

**Comment by egrath** [comment]
Exactly, that's the issue. The icon is shown in the bar and the tray even when the main window of the application is closed and only the tray icon should be there.

**Comment by tsic404** [comment]
> Exactly, that's the issue. The icon is shown in the bar and the tray even when the main window of the application is closed and only the tray icon should be there.

This may be not a issue, the app rewrite title and x. This should be the same in other DE.
